templates/apiary-hms-readonly.json

```diff
@@ -66,12 +66,21 @@
         "value": "${tcp_keepalive_probes}"
      }
     ],
-  "logConfiguration": {
-        "logDriver": "awslogs",
-        "options": {
+    "logConfiguration": {
+        "logDriver": "${enable_splunk_logging ? "splunk" : "awslogs"}",
```

Comment:
not a blocker but for future consider setting:
```
"logDriver" : "${logDriver}
"option" : "${logOptions} --> This needs to be jsonencoded somehow but I think that's possible.
```
Then we avoid the ifs and we can also introduce additional logging solutions.